lockfile_unix.go

```diff
@@ -161,7 +180,7 @@ func (l *lockfile) Unlock() {
 		// Close the file descriptor on the last unlock.
```

Comment:
A `unix.FcntlFlock()` call to clear the lock should be unnecessary if we're going to close the descriptor right after.

Reply:
That's unrelated to this PR. Let's fix that in another one.